pkg/postgres/pgadmin/admin_utils.go

```diff
@@ -0,0 +1,67 @@
+package pgadmin
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/stackrox/rox/pkg/logging"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+// DropDB - drops a database.
+func DropDB(sourceMap map[string]string, adminConfig *pgxpool.Config, databaseName string) error {
+	// Set the options for pg_dump from the connection config
+	options := []string{
+		"-f",
+		"--if-exists",
+		databaseName,
+	}
+
+	// Get the common DB connection info
+	options = append(options, GetConnectionOptions(adminConfig)...)
+
+	cmd := exec.Command("dropdb", options...)
+
+	SetPostgresCmdEnv(cmd, sourceMap, adminConfig)
+	err := ExecutePostgresCmd(cmd)
+	if err != nil {
+		log.Errorf("Unable to drop database %s", databaseName)
+		return err
+	}
+
+	return nil
+}
+
+// CreateDB - creates a database from template with the given database name
+func CreateDB(sourceMap map[string]string, adminConfig *pgxpool.Config, dbTemplate, dbName string) error {
+	// Set the options for pg_dump from the connection config
+	options := []string{
+		"-T",
+		dbTemplate,
+		dbName,
+	}
+
+	// Get the common DB connection info
+	options = append(options, GetConnectionOptions(adminConfig)...)
+
+	cmd := exec.Command("createdb", options...)
+
+	SetPostgresCmdEnv(cmd, sourceMap, adminConfig)
+
+	return ExecutePostgresCmd(cmd)
+}
+
+// RenameDB - renames a database
+func RenameDB(adminPool *pgxpool.Pool, originalDB, newDB string) error {
+	log.Infof("Renaming database %q to %q", originalDB, newDB)
+	sqlStmt := fmt.Sprintf("ALTER DATABASE %s RENAME TO %s", originalDB, newDB)
+
+	_, err := adminPool.Exec(context.TODO(), sqlStmt)
```

Comment:
in the future, what would we change this context to? Does having one with a timeout make sense?